packages/client/package.json

```diff
@@ -14,40 +14,55 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "bootstrap": "^4.6.0",
+    "bootstrap": "^4.6.1",
     "bootstrap-vue": "^2.19.0",
-    "core-js": "^3.15.0",
-    "dotenv": "^8.2.0",
+    "core-js": "^3.21.1",
+    "jquery": "3.6.0",
     "lodash": "^4.17.21",
     "lodash-checkit": "^2.3.3",
     "numeral": "^2.0.6",
-    "vue": "^2.6.11",
+    "popper.js": "1.16.1",
+    "vue": "^2.7.2",
+    "vue-good-table": "^2.21.11",
     "vue-multiselect": "^2.1.6",
-    "vue-router": "^3.5.1",
+    "vue-router": "^3.5.2",
     "vue-select": "^3.16.0",
     "vue2-filters": "^0.14.0",
     "vuelidate": "^0.7.6",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "@vue/cli": "~4.5.0",
-    "@vue/cli-plugin-babel": "~4.5.0",
+    "@typescript-eslint/eslint-plugin": "^5.26.0",
+    "@typescript-eslint/parser": "^5.26.0",
+    "@vue/cli": "^4.5.13",
+    "@vue/cli-plugin-babel": "~5.0.4",
     "@vue/cli-plugin-e2e-nightwatch": "~4.5.0",
```

Comment:
note to self: should probably bump version on `@vue/cli` and these other plugins to match `@vue/cli-service`